OpenOversight/app/templates/partials/officer_assignment_history.html

```diff
@@ -1,4 +1,10 @@
 <h3>Assignment History</h3>
+{% if officer.department.id == 1 %}
```

Comment:
This works, but just for posterity it might make sense to tie this to the department's short name rather than the database ID. Does this change seem okay?

```suggestion
{% if officer.department.short_name == 'SPD' %}
```

Reply:
Yup, that makes sense!